### DISCUSSION TITLE
Compiling for ESP32-S3-DevKitC-1-N8R8

### DISCUSSION THREAD

**Discussion by valdus2** [arendst/Tasmota#21168] [Q&A] [answered]
This is my first attempt compiling for such a module and I have some basic questions:

**1)** Is what [Jason2866](https://github.com/Jason2866) wrote, still true ?

Jason2866 Dec 6, 2022
@rug1024 You have to use this [board](https://github.com/arendst/Tasmota/blob/development/boards/esp32s3-qio_opi.json) to compile Tasmota
It is used in this env https://github.com/arendst/Tasmota/blob/development/platformio_tasmota_cenv_sample.ini#L33-L38
So you have to run platformio (after copying `platformio_tasmota_cenv_sample.ini` to `platformio_tasmota_cenv.ini`) with command `platformio run -e tasmota32s3-qio_opi-all`

_Originally posted by @Jason2866 in https://github.com/arendst/Tasmota/discussions/15935#discussioncomment-4321839_

**2)** What do I have to choose in platformio_override.ini ?

**3)** In user_config_override.h the changes are as usual ?

Thank you in advance.

**Comment by rug1024** [comment]
I use Tasmota-Gitpod to compile... https://gitpod.io/#https://github.com/arendst/Tasmota/tree/development

Changes in _platformio_override.ini_ not needed, just _user_config_override.h_

Compile with: **platformio run -e tasmota32s3-qio_opi-all** or **platformio run -e tasmota32s3-qio_opi-ard30** (for Arduino Core3)

**Comment by valdus2** [comment]
And what about changing `platformio_tasmota_cenv_sample.ini` to `platformio_tasmota_cenv.ini` ?
Is not needed anymore ?

Thank you.

**Reply by rug1024** [reply, marked as answer]
When I log into [Tasmota Github](https://gitpod.io/new/#https://github.com/arendst/Tasmota/tree/master) that file has already been created, but I don't change anything in there anymore.
Why don't you just give it a try?

**Comment by valdus2** [comment]
Thanks [rug1024](https://github.com/rug1024) for helping and [Jason2866](https://github.com/Jason2866) for confirming.
I asked, because when I last logged on on Gitpod at 13/4 (development tree), there was no `platformio_tasmota_cenv.ini` file created, only `platformio_tasmota_cenv_sample.ini`. Trying just now and it exists as both of you mentioned.
